framework/src/main/java/org/checkerframework/common/basetype/BaseTypeVisitor.java

```diff
@@ -1393,10 +1393,7 @@ public Void visitAssignment(AssignmentTree node, Void p) {
     @Override
     public Void visitEnhancedForLoop(EnhancedForLoopTree node, Void p) {
         AnnotatedTypeMirror var = atypeFactory.getAnnotatedTypeLhs(node.getVariable());
-        AnnotatedTypeMirror iterableType = atypeFactory.getAnnotatedType(node.getExpression());
-        AnnotatedTypeMirror iteratedType =
-                AnnotatedTypes.getIteratedType(
```

Comment:
AnnotatedTypes.getIteratedType is not longer called.  The Checker Framework still compiles if I delete it.  I think it should be removed.